__tests__/mint-tokens.test.js

```diff
@@ -15,7 +15,7 @@ describe('mint-tokens api', () => {
     const response = await TestUtils.request
       .post('/wallet/mint-tokens')
       .send({
-        token: '03',
+        token: '0000073b972162f70061f61cf0082b7a47263cc1659a05976aca5cd01b3351ee',
```

Comment:
Why change this?

Reply:
Because of a validation that happens in the wallet-lib regarding the token.